swagger.json

```diff
@@ -0,0 +1,190 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "version": "1.0.0", 
+      "title": "Tech Holding Classes",
+      "description": "An app to manage Tutorials"
+    },
+    "host": "localhost:8000",
+    "basePath": "/",
+    "tags": [
+      {
+        "name": "Users",
+        "description": "API for users in the system"
+      }
+    ],
+    "schemes": ["http"],
+    "consumes": ["application/json"],
+    "produces": ["application/json"],
+  
+
+  
+    "paths": {
+      "/users": {
```

Comment:
Can you please correct the paths of all the APIs? We're managing `tutorials` not `users`. Along with that please also correct `tag`, `summary` and other things which contains `user` keyword.